src/chainparams.cpp

```diff
@@ -80,7 +80,7 @@ static const Checkpoints::CCheckpointData dataRegtest = {
 
 class CMainParams : public CChainParams {
 public:
-    CMainParams() {
+    CMainParams(CScript scriptDestination) {
```

Comment:
Is MAIN ever used period? I think this should be there for symmetry's sake in any case.